tests/www/views/test_views_acl.py

```diff
@@ -190,11 +190,12 @@ def test_permission_exist(acl_app):
     perms_views = acl_app.appbuilder.sm.get_resource_permissions(
         acl_app.appbuilder.sm.get_resource('DAG:example_bash_operator'),
     )
-    assert len(perms_views) == 2
+    assert len(perms_views) == 3
 
     perms = {str(perm) for perm in perms_views}
     assert "can read on DAG:example_bash_operator" in perms
     assert "can edit on DAG:example_bash_operator" in perms
+    assert "can delete on DAG:example_bash_operator" in perms
```

Comment:
We probably need one more test as we have in https://github.com/apache/airflow/pull/20346/files#diff-81ef48c7266f1690897901984c78693b82eaef73c99d54c738782fc9541202ceR211-R219

to be sure, that User can:

a) delete a DAG if they just have `DAG.can_delete` 
b) delete a DAG if they just have `DAG:<dag_id>.can_delete` (and not `DAG.can_delete` )